docs/openapi-ts/migrating.md

```diff
@@ -48,6 +48,22 @@ This config option is deprecated and will be removed.
 
 This config option is deprecated and will be removed.
 
+## v0.42.0
+
+### Moved `operationId`
+
+This config option has been moved. You can now configure it using the `services.operationId` option.
```

Comment:
Can you update the other changes to use the same language please

Reply:
the other ones were removed (and added back as somethign new)., This was just moved without rename